Form-Control/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>My form exercise</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="styles.css">
+  <meta name="description" content="Order your custom t-shirt by choosing size, color, and preferred delivery date. Sign in to personalize your selection and complete your purchase easily.">
+
+
+
+</head>
+<body>
+  <header>
+    <h1>Your Tee, Your Way: Choose & Go!</h1>
+  </header>
+  <main>
+  <form id="tshirt-order-form">
+  <!---------------------------------------------- name ----------------------------------------------------------->
+    <div>
+      <label for="name">Name:</label>
+      <input type="text" name="name" id="name" placeholder="Enter your name" required minlength="2" required> <br>
+    </div>
+
+  <!---------------------------------------------- email ---------------------------------------------------------->
+    <div>
+      <label for="email">email:</label>
+      <input id="email" type="email" required name="email" placeholder="Enter your email">
+    </div>
+
+  <!---------------------------------------------- password ------------------------------------------------------->
+  <div>
+    <label for="password" >Password:
+      <input type="password" name="password" placeholder="password" required> <br>
+    </label>
+  </div>
+  <!---------------------------------------------- Delivery date -------------------------------------------------->
+  <div>
+    <label for="delivey-date" >Delivery date:
+      <input id="delivey-date" type="date" name="date" placeholder="delivey-date" required> <br>
+    </label>
+  </div>
+  <!---------------------------------------------- Size ----------------------------------------------------------->
+  <div>
+    <label for="size">Size</label>
+    <select name="size" id="size">
+      <option label="XS" value=" XS"> XS</option>
+      <option label="S" value="S">S</option>
+      <option label="M" value="M">M</option>
+      <option label="L" value="L">L</option>
+      <option label="XL" value="XL">XL</option>
+      <option label="XXL" value="XXL">XXL</option>
+    </select>
```

Comment:
Perfect. But I have a question, If I press Submit button by mistake without choosing my size, form would be submitted with the first Size (XS), How we can solve this problem?
How we can make it required option?

Reply:
hey @RihannaP! Great catch on that default size issue! Yeah, if someone hits "Submit" by accident, it could totally go through with "XS" even if they didn’t pick it. To fix that, I’ll add a “Select a size” option at the top that doesn’t actually count as a real choice. So, when they try to submit without picking an actual size, it’ll stop them with a little reminder.
  "<select name="size" id="size" required>
  <option value="" disabled selected>Select a size</option>"